classes/class-dintero-checkout-templates.php

```diff
@@ -79,20 +79,19 @@ public function override_template( $template, $template_name ) {
 			return $template;
 		}
 
+		// Dintero is not available for free orders except for free trial subscriptions. Refer to the Subscription class.
```

Comment:
Instead of just saying "Refer to Subscription class" maybe we should say what to look at? In this case I'm assuming its the filter we have for `woocommerce_cart_needs_payment` but that can be easy for people to miss.